### PR TITLE
Update run_on_flyte.rst

### PR DESCRIPTION
**Pull request by SandraGH5**
Edits to Run Your Workflow on a Flyte Backend

### CODE REVIEW

File: rsts/getting_started/run_on_flyte.rst [hunk 1]
```diff
@@ -25,50 +34,57 @@ Make Sure you have followed :ref:`getting-started-firstrun` and have a running d
...
-
-Register your workflows
------------------------
+Register Your Workflows
+=======================
+Registration is the process of shipping your code to the Flyte backend. This creates an immutable, versioned record of your code with the FlyteAdmin service.
 
-From within root directory of ``flyteexamples`` you created :ref:`previously <getting-started-first-example>`
-feel free to make any changes and then register ::
+From within root directory of ``flyteexamples`` you previously created with the :ref:`Write Your First Workflow tutorial <getting-started-first-example>`,
```
Comment (EngHabu) [inline rsts/getting_started/run_on_flyte.rst:44]: The* root directory?

**Review by EngHabu — APPROVED** [review]

File: rsts/getting_started/run_on_flyte.rst [hunk 2]
```diff
@@ -25,50 +34,57 @@ Make Sure you have followed :ref:`getting-started-firstrun` and have a running d
...
   FLYTE_AWS_SECRET_ACCESS_KEY=miniostorage make fast_register
 
 
-.. tip:: Flyte sandbox uses minio as a substitue for S3/GCS etc, in the first command we port-forwarded it to 30084. If you use s3/gcs or a different port-forward you can drop or change ``FLYTE_AWS_ENDPOINT`` accordingly.
+.. tip:: Flyte sandbox uses minio as a substitue for S3/GCS etc. It is port-forwarded in the first command to 30084. If you use S3/GCS or a different port-forward you can drop or change the ``FLYTE_AWS_ENDPOINT`` accordingly.
+
+.. rubric:: It's that simple!
 
-.. rubric:: Boom! It's that simple.
+Run Your Workflows
```
Comment (cosmicBboy) [inline rsts/getting_started/run_on_flyte.rst:55]: For the getting started guide, I think we should assume only one path... meaning that we assume the user has `http://localhost:30081/console` and not a custom `<host:port>`.

I suggest we get rid of the `K3d` and `Docker-desktop or other` headers and collapse this part into

```
Run Your Workflows
==================

Visit the page housing workflows registered to your project at:
`http://localhost:30081/console/projects/flyteexamples/workflows <http://localhost:30081/console/projects/flyteexamples/workflows>`__

Once you have accessed your workflows:
...
```

File: rsts/getting_started/run_on_flyte.rst [hunk 3]
```diff
@@ -25,50 +34,57 @@ Make Sure you have followed :ref:`getting-started-firstrun` and have a running d
...
-feel free to make any changes and then register ::
+From within root directory of ``flyteexamples`` you previously created with the :ref:`Write Your First Workflow tutorial <getting-started-first-example>`,
+feel free to make any changes and then register: ::
 
   FLYTE_AWS_ENDPOINT=http://localhost:30084/ FLYTE_AWS_ACCESS_KEY_ID=minio \
   FLYTE_AWS_SECRET_ACCESS_KEY=miniostorage make fast_register
 
 
-.. tip:: Flyte sandbox uses minio as a substitue for S3/GCS etc, in the first command we port-forwarded it to 30084. If you use s3/gcs or a different port-forward you can drop or change ``FLYTE_AWS_ENDPOINT`` accordingly.
+.. tip:: Flyte sandbox uses minio as a substitue for S3/GCS etc. It is port-forwarded in the first command to 30084. If you use S3/GCS or a different port-forward you can drop or change the ``FLYTE_AWS_ENDPOINT`` accordingly.
```
Comment (cosmicBboy) [inline rsts/getting_started/run_on_flyte.rst:51]: typp: `substitue -> substitute`

File: rsts/getting_started/run_on_flyte.rst (hunk 1 above)
Comment (SandraGH5) [reply rsts/getting_started/run_on_flyte.rst:44]: Replacing the word "previously" which was a link to the getting started example by "Write Your First Workflow tutorial"

**Review by cosmicBboy — APPROVED** [review]

**Pull request merged**